src/client/pythonEnvironments/discovery/locators/services/windowsStoreLocator.ts

```diff
@@ -167,4 +170,11 @@ export class WindowsStoreLocator extends Locator {
         }
         return undefined;
     }
+
+    private async registerWatchers(): Promise<void> {
```

Comment:
```suggestion
    private async startWatcher(): Promise<void> {
```

Also, why is the method async?

Reply:
The method returns a promise so that I can start the watcher in background without blocking the locator.
The method is `async` because Typescript complains in case of `private registerWatchers(): Promise<void>`